cypress/e2e/spec.cy.js

```diff
@@ -0,0 +1,6 @@
+describe('work&co headline check', () => {
+  it('passes', () => {
```

Comment:
Few minor changes:
1. `it('passes)` -> change 'passes' into a short description of the test step
2. instead of using class name(s) for selector, use the 'data-test-id' attribute since that one is unique and created exactly for automated tests
3.  Update README.md file with short info on how to install and run the tests.
And one major:
I don't see the `package.json` file in the files changes - and without that file, we don't know the dependencies  to install, meaning if anyone clones your repo - he can't instal & run the cypress.
(also package-lock.json, it's a good practice to push that file as well)